internal/internal_nexus_worker.go

```diff
@@ -61,19 +62,24 @@ func newNexusWorker(opts nexusWorkerOptions) (*nexusWorker, error) {
 		params,
 	)
 
+	workerType := "NexusWorker"
+	metricsHandler := params.MetricsHandler.WithTags(metrics.WorkerTags(workerType))
+	tss := newTrackingSlotSupplier(NewFixedSizeSlotSupplier(opts.executionParameters.ConcurrentNexusTaskExecutionSize), metricsHandler)
```

Comment:
Is there a reason we didn't support Nexus.

Reply:
Yeah, I wanted to have at least some discussion about whether or not we want to add it to the tuning interface. Probably the answer is yes but it's worth discussing. If we decide yes it's easy enough to add subsequently.